cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -408,6 +408,14 @@ data:
             proxy_set_header  X-Real-IP  $remote_addr;
             proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
         }
+        location = /model/allocation/autocomplete {
+            proxy_read_timeout          300;
+            proxy_pass http://aggregator/allocation/autocomplete;
+            proxy_redirect off;
+            proxy_set_header Connection "";
+            proxy_set_header  X-Real-IP  $remote_addr;
+            proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+        }
```

Comment:
TODO pre-merge: Add caching

Reply:
cache with nginx? need any help? 

https://github.com/kubecost/cost-analyzer-helm-chart/commit/d75698466d187a83c92a42a6a10e91357653073d

note: we should set the size to a reasonable number, 500m in the example above is likely way overkill.